src/customElements.ts

```diff
@@ -65,12 +65,12 @@ export interface CustomElementInitializer {
  *
  * Describes a custom element.
  *
- * @property    {string}                                tagName         	The tag name to register this widget under. Tag names must contain a "-"
- * @property    {WidgetConstructor}                     widgetConstructor   A widget Constructor that will return the widget to be wrapped in a custom element
- * @property    {CustomElementAttributeDescriptor[]?}   attributes     		A list of attributes to define on this element
- * @property    {CustomElementPropertyDescriptor[]?}    properties      	A list of properties to define on this element
- * @property    {CustomElementEventDescriptor[]?}       events          	A list of events to expose on this element
- * @property    {CustomElementInitializer?}             initialization  	A method to run to set custom properties on the wrapped widget
+ * @property tagName             The tag name to register this widget under. Tag names must contain a "-"
```

Comment:
Probably a silly question, but should tagName and widgetConstructor be listed here when they are not listed in the interface?